pkgs/applications/editors/mg/default.nix

```diff
@@ -1,5 +1,7 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, ncurses, buildPackages }:
-
+let
```

Comment:
Not required

Reply:
Do you mean moving `isCross` to arguments?

Reply:
I think moving it inline (so `lib.optionalString stdenv.build...`).